src/ape_plugins/_cli.py

```diff
@@ -22,38 +23,80 @@ def cli():
     """
 
 
-@cli.command(name="list", short_help="List installed plugins")
+def _format_output(plugins_list: List[str], version: Optional[str]) -> Set:
+    output = set()
+    prefix = "ape_"
+    for i in plugins_list:
+        if i.startswith(prefix):
```

Comment:
Sorry I didn't see this before either!:
But lines 30-32 can be replaced with just `i.strip("ape_")` and you don't have to define prefix at line 28 either then

Reply:
Talked offline, but lines 30 - 34 should just be replaced with `text = i.replace("ape_", "")`